app/uk.gov.hmrc.helptosaveproxy/config/AppConfig.scala

```diff
@@ -40,4 +40,7 @@ object AppConfig extends ServicesConfig {
 
   val nsiCreateAccountUrl: String = s"${baseUrl("nsi")}/nsi-services/account"
 
+  def dwpUCClaimantCheckUrl(nino: String): String =
```

Comment:
name is difficult to read, can be simplified to  `dwpUrl`

Reply:
Done, good point!